app/server/locale/ussd.en.yml

```diff
@@ -53,17 +59,24 @@ en:
       2. Cancel
       0. Back
     directory_listing: |
-      CON Choose Market Category
-      1. Food
-      2. Water
-      3. Energy
-      4. Education
-      5. Health
-      6. General shop
-      7. Environment
-      8. Transport
-      9. Labour
+      CON Choose Market Category 
+      %{options}
+      9. Show other options
     complete: END Your request has been sent. You will receive an SMS shortly.
+    directory_listing_other:
+      first: |
+        CON Choose Market Category 
+        %{other_options}
+        9. Show other options
```

Comment:
swahili lists 0. Back as an option, but english doesn't

Reply:
I dunno tbh. It was like that before I started working on it. But I think it does make sense to add it

Reply:
I'm adding the back button